src/components/Pane.jsx

```diff
@@ -0,0 +1,47 @@
+var React = require('react'),
+    classNames = require('classnames'),
+    style = require('./component-playground.less');
+
+export default React.createClass({
+  displayName: 'Pane',
+
+  propTypes: {
+    split: React.PropTypes.oneOf(['vertical', 'horizontal']),
+    children: React.PropTypes.object.isRequired
+  },
+
+  getInitialState: function() {
+    return {};
+  },
+
+  render: function() {
+    var classes = {};
+    classes[style[this.props.split]] = true;
+    classes[style['pane']] = true;
+    classes = classNames(classes);
+
+    var split = this.props.split || 'vertical';
```

Comment:
It doesn't seem like you actually need this since you do `if (split === 'horizontal') {} else {}` below.